### PR TITLE
feat: add features section component

### PR DESCRIPTION
**Pull request by erdemkarakas**
**What was done**

Features component added 

`constants.ts` listed icons added 

In `Features.ts` we used the `<featureIcon.component>` definition for icons.

TODO:

- [ ] Includes temporarily used icons from react-feather until svgr was added

### CODE REVIEW

File: components/Features/Features.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+      <div className="grid grid-cols-4 grid-rows-4 gap-4 px-8 md:grid-cols-8  md:grid-rows-2 md:gap-8">
+        {Icons.map(featureIcon => (
+          <div
+            key={featureIcon.id}
+            className="flex items-center justify-center"
+          >
+            <featureIcon.component
+              className="h-16 w-16 md:h-20 md:w-20"
+              color={'#383838'}
```
Comment (eraygundogmus) [inline components/Features/Features.tsx:29]: you can remove curlies, i guess

File: components/Features/Features.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            />
+          </div>
+        ))}
+      </div>
+
+      <div className="flex items-center justify-center">
+        <Button size={'xl'}>View all integrations</Button>
```
Comment (eraygundogmus) [inline components/Features/Features.tsx:36]: you can remove curlies, i guess

File: components/Features/Features.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        <h3 className="mt-3 text-center text-3xl font-semibold text-gray-900 md:text-4xl">
+          Get more value from your tools
+        </h3>
+        <p className="mt-4 text-center  text-lg font-normal text-gray-500 md:mt-5 md:text-xl">
+          Connect your tools, connect your teams. With over 200 apps already
+          available in our directory, your team's favourite tools are just a
+          click away.
+        </p>
+      </div>
+
```
Comment (eraygundogmus) [inline components/Features/Features.tsx:20]: Can you split this part?

File: components/Features/Features.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import Button from './../Button/Button';
+import { Icons } from './constants';
```
Comment (eraygundogmus) [inline components/Features/Features.tsx:2]: I don't think Icon is a constant. It is an asset.

**Review by eraygundogmus — CHANGES_REQUESTED** [review]

File: components/Features/Features.tsx (hunk 3 above)
Comment (erdemkarakas) [reply components/Features/Features.tsx:20]: Do you mean to do with "Compound Components" or split with the "<div>" tag?

File: components/Features/Features.tsx (hunk 4 above)
Comment (erdemkarakas) [reply components/Features/Features.tsx:2]: @oguzergul  It think said that like it, It's going to be change

File: components/Features/Features.tsx (hunk 3 above)
Comment (eraygundogmus) [reply components/Features/Features.tsx:20]: It does not have to be **compound**. I think you can add it as a JSX function too for just splitting list logic from other

**Review by oguzergul — CHANGES_REQUESTED** [review]
Component names should be PascalCase. You need to change as Feature.Component.

File: components/Features/Features.tsx (hunk 4 above)
Comment (oguzergul) [reply components/Features/Features.tsx:2]: I did not understand your reply @erdemkarakas what did I say?

File: components/Features/constants.ts [hunk 5]
```diff
@@ -0,0 +1,73 @@
+import { Icon, Linkedin, Slack } from 'react-feather';
+
+interface IIcon {
```
Comment (oguzergul) [inline components/Features/constants.ts:3]: These are not icons. You need to replace these. These are svg brand logos.[This is a design file](https://www.figma.com/file/O3IvMDpBVUBMzEU6ZuRdsX/Frontenship-Website?node-id=4%3A14026&t=fiILYcZsyTIptJtU-0)

File: components/Features/constants.ts [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  id: number;
+  component: Icon;
+}
+
+export const Icons: Array<IIcon> = [
+  {
+    id: 1,
+    component: Slack
```
Comment (oguzergul) [inline components/Features/constants.ts:11]: When you replace icons with svg. You can return these brand logos like `Icon: () => <JavaScriptLogo/>`

**Review by oguzergul — CHANGES_REQUESTED** [review]

**Review by eraygundogmus — APPROVED** [review]

**Pull request merged**